pandas/tests/io/parser/test_usecols.py

```diff
@@ -21,6 +21,20 @@
                                "expected but not found: {0}")
 
 
+@pytest.mark.parametrize("names,usecols", [
+    (None, [0, 3]),
+    (["a", "b", "c"], [0, -1, 2]),
+    (None, [3]),
+    (["a"], [3])
+])
+def test_usecols_out_of_bounds(all_parsers, names, usecols):
+    data = "a,b,c\n1,2,3\n4,5,6"
+    parser = all_parsers
+
+    with pytest.raises(ValueError, match=_msg_validate_usecols_names):
```

Comment:
although this technically doesn't fail the regex, it appears that `_msg_validate_usecols_names` is intended to be used with `.format()`

Reply:
Yea good catch @simonjayhawkins

Reply:
indeed! @jreback I'll push an update this weekend